src/sockets/sockets.ts

```diff
@@ -1033,13 +1038,14 @@ export const server = function (io: SocketServer): void {
       }
 
       if (
-        socket.request.user.avatarImgRes &&
-        !socket.request.user.avatarImgRes.includes('proavalon.com') ||
-        socket.request.user.avatarImgSpy &&
-        !socket.request.user.avatarImgSpy.includes('proavalon.com')
+        (socket.request.user.avatarImgRes &&
+          !socket.request.user.avatarImgRes.includes('proavalon.com')) ||
+        (socket.request.user.avatarImgSpy &&
+          !socket.request.user.avatarImgSpy.includes('proavalon.com'))
```

Comment:
Can you make a smaller PR and get this in

Reply:
Yep, done in https://github.com/vck3000/ProAvalon/pull/673